Core/include/Acts/Seeding/HoughTransformUtils.hpp

```diff
@@ -109,23 +109,33 @@ class HoughCell {
   YieldType nLayers() const { return m_nLayers; }
   /// @brief access the number of unique hits compatible with this cell
   YieldType nHits() const { return m_nHits; }
-  /// @brief access the set of layers compatible with this cell
-  const std::unordered_set<unsigned>& layers() const { return m_layers; }
-  /// @brief access the set of unique hits compatible with this cell
-  const std::unordered_set<identifier_t>& hits() const { return m_hits; }
+  /// @brief access the span of layers compatible with this cell
+  std::span<const unsigned, std::dynamic_extent> getLayers() const;
+  // /// @brief access the pan of unique hits compatible with this cell
+  std::span<const identifier_t, std::dynamic_extent> getHits() const;
+
   /// @brief reset this cell, removing any existing content.
   void reset();
 
  private:
   /// data members
 
-  YieldType m_nLayers =
-      0;                  // (weighted) number of layers with hits on this cell
-  YieldType m_nHits = 0;  // (weighted) number of unique hits on this cell
-  std::unordered_set<unsigned> m_layers =
-      {};  // set of layers with hits on this cell
-  std::unordered_set<identifier_t> m_hits =
-      {};  // set of unique hits on this cell
+  YieldType m_nLayers{0};  // (weighted) number of layers with hits on this cell
+  YieldType m_nHits{0};    // (weighted) number of unique hits on this cell
```

Comment:
```suggestion
  /// (weighted) number of layers with hits on this cell
  YieldType m_nLayers{0};
  /// (weighted) number of unique hits on this cell
  YieldType m_nHits{0};
```
- `///` for doc
- to me it is more readable to put the comment above